src/pages/calculator.tsx

```diff
@@ -0,0 +1,328 @@
+												type="radio"
+												id="shape"
+												name="shape"
+												value="rectangle"
+												onChange={onInputChange}
+											/>
+											Rectangular surface
+										</FormRadioLabel>
+									</FormRadioGroup>
+
+									<FormRadioGroup>
+										<FormRadioLabel>
+											<input
+												type="radio"
+												id="shape"
+												name="shape"
+												value="cylinder"
+												onChange={onInputChange}
+											/>
+											Round surfaces and Cylinders
+										</FormRadioLabel>
+									</FormRadioGroup>
+								</FormRadioWrapper>
+
+								<ParagraphText>
+									What are the dimensions of your project?
+								</ParagraphText>
+								<FormInputWrapper>
+									{shape === 'rectangle' ? (
+										<>
+											<FormInput
+												min="0"
+												type="number"
+												id="length"
+												name="length"
+												value={length}
+												placeholder="Length"
+												onChange={onInputChange}
+											/>
+											<FormInput
+												type="number"
+												id="width"
+												name="width"
+												value={width}
+												placeholder="Width"
+												onChange={onInputChange}
+											/>
+										</>
+									) : (
+										<FormInput
+											type="number"
+											id="diameter"
+											name="diameter"
+											value={diameter}
+											placeholder="Diameter"
+											onChange={onInputChange}
+										/>
+									)}
+
+									<FormInput
+										type="number"
+										id="thickness"
+										name="thickness"
+										value={thickness}
+										placeholder="Coating Thickness"
+										onChange={onInputChange}
+									/>
+								</FormInputWrapper>
+
+								<FormSelect
+									onChange={onInputChange}
+									name="unit"
+									id="unit"
+									defaultValue={'default'}
+									placeholder="Choose your unit">
+									<option disabled value="default">
+										Choose your unit
+									</option>
+									<option value="CM">Centimeter (CM)</option>
+									<option value="M">Meter (M)</option>
+									<option value="In">Inches (In)</option>
+									<option value="Ft">Feet (Ft)</option>
+								</FormSelect>
+
+								<StyledCTA className="no-animate" type="submit">
+									Calculate
+								</StyledCTA>
+							</form>
+						</CalculateContent>
+						<ResultsContent>
+							<h3>Preview Final Amount</h3>
+							<p>Here is the amount of resin you will need for your project.</p>
+							<FinalAmount data={dataOne} />
+							<FinalAmount data={dataTwo} />
+						</ResultsContent>
+					</CalculatorWrapper>
+				</PageContainer>
+
+				<PageContainer>
+					<ResinProducts>Make</ResinProducts>
+				</PageContainer>
+			</PageWrapper>
+		</>
+	);
+}
+
+export default Calculator;
+
+const PageWrapper = styled.div`
+	background-color: var(--oex-lightest-grey);
+	margin-top: 5rem;
+	padding-top: 7rem;
+
+	@media (min-width: 768px) {
```

Comment:
start using the styled-component themed media query eg.  `@media ${({theme})=>theme.breakpoints.above.md}`

Reply:
Ok. Done